modules/kubernetes/goldpinger/templates/values.yaml.tpl

```diff
@@ -0,0 +1,7 @@
+goldpinger:
+  port: 8080
+
+podAnnotations:
+  linkerd.io/inject: enabled
```

Comment:
Add a if statment check if linkerd enabled or not

Reply:
Done